packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Title.tsx

```diff
@@ -61,16 +62,21 @@ const ComponentContent = (props: ComponentContentProps) => {
 export const Default = (props: TitleProps): JSX.Element => {
   const datasource = props.fields?.data?.datasource || props.fields?.data?.contextItem;
   const { sitecoreContext } = useSitecoreContext();
+  const isMetadataMode = sitecoreContext?.editMode === 'metadata';
 
   const text: TextField = {
     value: datasource?.field?.jsonValue?.value,
-    editable: datasource?.field?.jsonValue?.editable,
+    ...(isMetadataMode
+      ? { metadata: props?.fields?.data?.contextItem?.field?.jsonValue?.metadata }
+      : { editable: datasource?.field?.jsonValue?.editable }),
```

Comment:
Why are we going to `contextItem` for `metadata` when we are using `datasource` for `editable` (and `value`)?